cmd/kms-rest/startcmd/start.go

```diff
@@ -688,6 +703,8 @@ func startKmsService(params *kmsRestParameters, srv Server) error {
 		router.HandleFunc(handler.Path(), handler.Handle()).Methods(handler.Method())
 	}
 
+	go startMetrics(srv, params.hostMetricsURL)
```

Comment:
Can we put the if condition from https://github.com/trustbloc/kms/pull/204/files#diff-71b59fb90584bba92a458b851f2d3f6d1aa7829f99f30cb881fb3b43a50294a9R725 (the one inside the startMetrics function) to here ? We can avoid creating go routine if its not required.